src/client/Microsoft.Identity.Client/MsalServiceExceptionFactory.cs

```diff
@@ -11,6 +11,8 @@ namespace Microsoft.Identity.Client
 {
     internal class MsalServiceExceptionFactory
     {
+        private const string STSTooManyCallsError = "AADSTS50196";
+
```

Comment:
nit: `AadThrottledErrorCode`

can you move this to the constants class (i think there is one) so you can reuse in the test?

Reply:
Good suggestion. Done